virtcontainers/qemu_ppc64le.go

```diff
@@ -105,12 +105,13 @@ func (q *qemuPPC64le) cpuModel() string {
 
 func (q *qemuPPC64le) memoryTopology(memoryMb, hostMemoryMb uint64) govmmQemu.Memory {
 
-	if hostMemoryMb > defaultMemMaxPPC64le {
-		hostMemoryMb = defaultMemMaxPPC64le
-	} else {
+	if qemuMajorVersion >= 2 && qemuMinorVersion >= 10 {
```

Comment:
I'd consider adding a couple of logging calls here, one for each scenario so it's clear what's happening from the log.

What we cannot reliably check the guest kernel version so maybe your debug entry for handling qemu 2.10+ should mention the 4.11 guest kernel assumption.

Reply:
@jodh-intel : Done.  Updated my PR.